.github/workflows/sync-package-periodically.yml

```
@@ -0,0 +1,45 @@
+name: Sync Package Metadata
+
+on:
+  schedule:
+    - cron: '0 0 * * 6'
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    
+    env:
+     Codeql.Enabled: true
+     Enable_SyncPackage: false
+     
```

Where are these variables used?

---

Enable_SyncPackage is not in scope and removed it.  The Codeql.Enabled setting is required as part of Microsoft compliance check.

---

```suggestion
     Codeql.Enabled: true
```
If it isn't being used we should remove it.